components/seo/JsonLd.tsx

```
@@ -0,0 +1,92 @@
+        mainEntity: data?.questions || [],
+      }
+      break
+
+    default:
+      structuredData = baseStructuredData
+  }
+
+  return (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify(structuredData) }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)